### DISCUSSION TITLE
I get a Json error when trying to login

### DISCUSSION THREAD

**Discussion by Seterina** [nilsteampassnet/TeamPass#3623] [Q&A]
I have Teampass 3.0.0.23 mounted on Linux and although LDAP works and users are apparently created fine, when any user tries to access Teampass I get a Json error.

Does anybody know what is it due to?

![image](https://user-images.githubusercontent.com/129095272/228006771-e9cec1a5-178a-43dc-a545-488827906369.png)

Thank you so much.

**Comment by nilsteampassnet** [comment]
Upgrade to latest version and also please share errors in web console

**Comment by Seterina** [comment]
Hello,

Sorry for the delay in replying to this email, for now I managed to solve
it.

Thank you very much for your dedication.

Kind regards

Marta

Missatge de Nils Laumaillé ***@***.***> del dia dl., 10
d’abr. 2023 a les 8:33:

> Upgrade to latest version and also please share errors in web console
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nilsteampassnet/TeamPass/discussions/3623#discussioncomment-5568187>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6Y5M2G2KZVG4JLZDJ45SV3XAOSUFANCNFSM6AAAAAAWJMGT5E>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Marta Jiménez Pino
***@***.*** ***@***.***> *
